configs/ot2_defconfig

```diff
@@ -43,12 +43,12 @@ BR2_PACKAGE_RPI_USERLAND=y
 BR2_PACKAGE_UBOOT_TOOLS=y
 BR2_PACKAGE_USBMOUNT=y
 BR2_PACKAGE_PYTHON3=y
-BR2_PACKAGE_PYTHON3_BZIP2=y
```

Comment:
We want this still I think... savedefconfig keeps deleting it for some reason but for now let's manually re-add it

Reply:
Ahh, yeah there do seem to be some unexpected changes. I'll fix that.